backend/config.go

```diff
@@ -150,6 +212,9 @@ func (cfg *Config) Connect(replicas []ReplicaInfo) (err error) {
 		replica.node = node
 	}
 
+	cfg.connected = true
+	cfg.mods.EventLoop().AddEvent(connected{})
```

Comment:
Would be nice with a comment referring back to the `connected{}` event handler, and what it will do on this event.